core/common/src/IModelError.ts

```diff
@@ -32,6 +32,43 @@ export class IModelError extends BentleyError {
   }
 }
 
+/** Detailed information about a particular object Lock that is causing the Lock update conflict.
+ * @public
+*/
+export interface ConflictingLock {
+  /** Id of the object that is causing conflict. */
+  objectId: string;
+  /**
+     * The level of conflicting lock. Possible values are {@link LockLevel.Shared}, {@link LockLevel.Exclusive}.
+     * See {@link LockLevel}.
+     */
+  lockLevel: LockLevel;
+  /** An array of Briefcase ids that hold this lock. */
+  briefcaseIds: number[];
+}
+
+/** Supported Lock levels.
```

Comment:
Link to `LockControl` or ConcurrencyControl.md so people know what "Lock" means.

Reply:
Linked to ConcurrencyControl.md, specifically the section about how to acquire a lock